pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraItems.scala

```diff
@@ -114,4 +93,31 @@ case class SierraItems(itemDataMap: Map[SierraItemNumber, SierraItemData])
       None
     }
   }
+
+  private def transformItemData(
+    itemId: SierraItemNumber,
+    itemData: SierraItemData): Identifiable[Item] = {
+    debug(s"Attempting to transform $itemId")
```

Comment:
Want to keep this?

Reply:
This was from the original code (which was copy / pasted), I'm not that bothered about it myself but could be useful